NWN.Anvil/src/main/API/Object/NwCreature.cs

```diff
@@ -1333,6 +1335,15 @@ public void ClearDamageLevelOverride()
       DamageLevelOverrideService.Value.ClearDamageLevelOverride(this);
     }
 
+    /// <summary>
+    /// Clears the modifier that is set for the creature's initiative.<br/>
+    /// </summary>
+    public void ClearInitiativeModifier()
+    {
+      InitiativeModiferService.Value.ClearInitiativeModifer(this);
```

Comment:
Minor nit - `modifier` spelling is inconsistent in NwCreature & the service.

I've marked the other locations below.